theories/dune.disabled

```diff
@@ -7,6 +7,7 @@
  ; (mode native)
  (boot)
  (plugins
+   coq-core.plugins.ltac2_ltac1
```

Comment:
Shouldn't it be in the Ltac2 dune file?

Reply:
Indeed, thanks! I misunderstood the purpose of that plugin.